components/kubeflow/kfserving/src/kfservingdeployer.py

```diff
@@ -34,103 +34,231 @@
 from kfserving import V1alpha2InferenceServiceSpec
 from kfserving import V1alpha2InferenceService
 
+
 def EndpointSpec(framework, storage_uri):
```

Comment:
You are missing the `service_account` argument here?

Reply:
So true, I was a little sloppy. I fix